stretch_moveit2/stretch_moveit_config/src/moveit_draw.cpp

```diff
@@ -140,17 +140,17 @@ int main(int argc, char** argv)
     RCLCPP_INFO(LOGGER, "Visualizing plan (joint space goal) %s", success ? "" : "FAILED");
 
     if(i <= 1){
-      disp_plan.trajectory = my_plan.trajectory;
+      disp_plan.trajectory_ = my_plan.trajectory_;
```

Comment:
These changes are unrelated to this PR and are probably not necessary. Consider raising a separate PR?